climada/entity/exposures/crop_production.py

```diff
@@ -142,19 +155,38 @@ def set_from_single_run(self, input_dir=INPUT_DIR, filename=None, hist_mean=HIST
                 f.i. 'mai', 'ric', 'whe', 'soy'
             irr (string): irrigation type
                 f.i 'firr' (full irrigation), 'noirr' (no irrigation) or 'combined'= firr+noirr
+            isimip_version(str): 'ISIMIP2' (default) or 'ISIMIP3'
             unit (string): unit of the exposure (per year)
-                f.i 'USD' or 't'
+                f.i 'USD' or 't' (default) or 'kcal'
             fn_str_var (string): FileName STRing depending on VARiable and
                 ISIMIP simuation round
 
         Returns:
             Exposure
         """
-
+        if not bbox:
+            bbox = BBOX
+        if not input_dir:
+            input_dir = INPUT_DIR
+        if hist_mean is None:
+            hist_mean = HIST_MEAN_PATH
+        if not fn_str_var:
+            fn_str_var = FN_STR_VAR
+        if (not isimip_version) or ('ISIMIP2' in isimip_version):
+            isimip_version = 'ISIMIP2'
+        elif 'ISIMIP3' in isimip_version:
+            isimip_version = 'ISIMIP3'
+        if (not scenario) or ('hist' in scenario):
```

Comment:
Why this strange or clause here?

Reply:
their are different variants of scenario's that all require the same choice here, how about:

` if (not scenario) or (scenario in ('historical', 'hist')):`